maintainers/maintainer-list.nix

```diff
@@ -16564,6 +16564,12 @@
     github = "rubyowo";
     githubId = 105302757;
   };
+  rucadi = {
```

Comment:
Welcome 👋🙂

Reply:
Please add yourself to maintainer-list as separate commit.